packages/cozy-procedures/src/components/overview/DocumentsFullyCompleted.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { InlineCard, Icon } from 'cozy-ui/transpiled/react'
+const DocumentsFullyCompleted = ({ documents, navigateTo }) => {
+  const flatenedFiles = []
+  Object.values(documents).map(document => {
+    Object.values(document['files']).map(file => {
+      flatenedFiles.push(file)
+    })
+  })
+  return (
+    <InlineCard
+      className="u-c-pointer u-p-1 u-flex u-flex-column"
+      onClick={() => navigateTo('documents')}
+    >
+      {flatenedFiles.map((file, index) => {
+        let style = 'u-ellipsis u-flex'
+        if (index !== 0) style += ' u-mt-1'
```

Comment:
Why do we need a top margin on the first element, since there's a padding on the wrapper?

Reply:
This is because we don't need a top margin on the first element ;)

Reply:
Oh right, I misread that. Nit, but we use single directional margins whenever possible and usually it's towards the bottom, so using bottom margins would be more appropriate. Or given the layout, a small top and bottom padding might be even better!

Reply:
Will fix it soon thanks for the feedback